DICOMPlugins/DICOMTID1500Plugin.py

```diff
@@ -86,8 +102,86 @@ def referencedSeriesName(self, loadable):
     if hasattr(loadable, "referencedSOPInstanceUID"):
       referencedName = self.defaultSeriesNodeName(loadable.referencedSOPInstanceUID)
     return referencedName
-
+  
+  def getDICOMTagValue(self, tag_name):
+    """This function takes as input DICOM tag name, and 
+      returns the numeric string for that name"""
+      
+    if tag_name == "PatientID":
+      tag_numeric = "0010,0020"
+    elif tag_name == "StudyDate":
+      tag_numeric = "0008,0020"
+    elif tag_name == "StudyInstanceUID":
+      tag_numeric = "0020,000D"
+    elif tag_name == "SeriesInstanceUID": 
+      tag_numeric = "0020,000E" 
+    elif tag_name == "SeriesNumber":
+      tag_numeric = "0020,0011"
+    elif tag_name == "SeriesDescription":
+      tag_numeric = "0008,103E"
+    elif tag_name == "SOPInstanceUID": 
+      tag_numeric = "0008,0018"
+    elif tag_name == "ReferencedSeriesSequence":
+      tag_numeric = "0008,1115"
+    elif tag_name == "Modality":
+      tag_numeric = "0008,0060"
+    elif tag_name == "FrameOfReferenceUID": 
+      tag_numeric = "0020,0052"
+    elif tag_name == "PixelSpacing":
+      tag_numeric = "0028,0030"
+    elif tag_name == "Rows":
+      tag_numeric = "0028,0010"
+    elif tag_name == "Columns": 
+      tag_numeric = "0028,0011"
+    else:
+      tag_numeric = "" 
+    
+    return tag_numeric
+  
+  def getSRCode(self, tag_name):
```

Comment:
I would rename "tag_name" to "code_meaning" or something like that - the current parameter name can be misleading.